db/migration/V1.0__tables_migration.sql

```diff
@@ -1,6 +1,6 @@
 create table if not exists `family` (
 `id` bigint not null auto_increment primary key,
-`surname` varchar(10),
+`surname` varchar(20),
```

Comment:
increase number of characters in another file. Imagine that all developers already created the table. You should modify table in another flyway file.